api/api_sources/sources/database/database-schema/schema-sqls/ObservationSchema.sql

```diff
@@ -11,6 +11,16 @@ ALTER TABLE observation ADD COLUMN area_length NUMERIC(7, 2) NULL DEFAULT 0.0;
 ALTER TABLE observation ADD COLUMN access_description VARCHAR(500) NULL;
 ALTER TABLE observation ADD COLUMN observer_first_name VARCHAR(100) NULL;
 ALTER TABLE observation ADD COLUMN observer_last_name VARCHAR(100) NULL;
+ALTER TABLE observation ADD COLUMN sample_identifier VARCHAR(50) NULL;
+ALTER TABLE observation ADD COLUMN range_unit_number VARCHAR(50) NULL;
```

Comment:
We should be careful with allowing this as a free form field... RANGE UNIT numbers come from another Ministry table called RANGE_UNIT. Rather than have users put whatever they want in the column, we should be pulling from pre-existing data. This should probably be looked into further...

Reply:
Example of where this data is in the BCGW... https://catalogue.data.gov.bc.ca/dataset/range-units

Reply:
I created task[https://apps.nrs.gov.bc.ca/int/jira/browse/INS-579] to work on this issue.